# This message encodes Triangle mesh message

std_msgs/Header header
int32 id
mesh_msgs/TriangleMesh mesh